### ISSUE TITLE
Unable to login to gmail

### ISSUE THREAD

**Issue by iankellogg** [daskeyboard/daskeyboard-applet--gmail#4]
Unable to log into the gmail prompt due to "This browser or app may not be secure."

I've tried this on two different windows 10 computers
![gmailfail](https://user-images.githubusercontent.com/19454690/74242024-2c070e80-4cab-11ea-9830-815850aec320.png)

**Comment by Googrosh** [comment]
Same here!
I also tried "Find my account" button (I tried to avoid somehow the login page) that guides you through several steps and sends a verification code on your smartphone but, in the end, same result.
I hope someone handles this issue.

**Comment by Architekt909** [comment]
Same issue with the warning about the browser not being secure. Windows 10x64

**Comment by everystone** [comment]
Just ran into this as well. 
```Sign in with Google temporarily disabled for this app. This app has not been verified yet by Google in order to use Google Sign In.```

Doesn't look like these applets are maintained at all, kinda sad when its the selling point of the Q keyboards.

**Comment by Googrosh** [comment]
> Just ran into this as well. `Sign in with Google temporarily disabled for this app. This app has not been verified yet by Google in order to use Google Sign In.`
> 
> Doesn't look like these applets are maintained at all, kinda sad when its the selling point of the Q keyboards.

Indeed! I just deactivate any applet on my keyboard. I bought it for the applets but now I am using it as a normal keyboard... Waste of money!
